taxi/tests/test_views.py

```diff
@@ -0,0 +1,36 @@
+from django.test import TestCase
+from django.urls import reverse
+
+from taxi.models import Manufacturer, Driver
+
+
+class TaxiIndexViewTest(TestCase):
+    def test_index_view(self):
+        response = self.client.get(reverse("taxi:index"))
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, "taxi/index.html")
+
+
+class ViewsTest(TestCase):
+    def setUp(self):
+        self.user = Driver.objects.create_user(
+            username="testuser", password="12345"
+        )
+
+    def test_manufacturer_list_view(self):
```

Comment:
Looks like you wanted to test here if user can see page if he is authenticated
You can just check status_code then. Also you should check if user has access to the page if he is not authenticated